kubespawner/spawner.py

```diff
@@ -274,6 +291,32 @@ def _hub_connect_port_default(self):
         """
     )
 
+    storage = Unicode(
+        "1Gi",
+        config=True,
+        help="""
+        The ammount of storage space to request from the volume that the pvc will
+        mount to.
+        """
+    )
+
+    storage_class = Unicode(
```

Comment:
user_storage_class? Same note re: None. In general use None to mean 'ignore this setting' rather than reusing ''.